guides/common/modules/proc_mirroring-remote-flatpak-repositories-to-project-products.adoc

```diff
@@ -12,10 +12,20 @@ Flatpak repositories are container repositories and you can add them to content
 Flatpak applications require a corresponding runtime environment, which the Flatpak remote also provides. 
 To make the {EL} 10 Mozilla Firefox Flatpak available to a host, ensure that the host can access the matching `rhel10/flatpak-runtime` repository.
 
+.Prerequisites
+* Your {Project} account has a role that grants the `create_flatpak_remotes` permission. 
+ 
+ * Your {Project} account has the `view_flatpak_remotes`, `edit_flatpak_remotes`, and `destroy_flatpak_remotes` permissions to view, modify, or delete Flatpak remotes. 
+
+* Flatpak applications rely on Flatpak runtimes. 
+For example, `rhel9/firefox-flatpak` depends on `rhel9/flatpak-runtime`.
+
+* Ensure that runtime repositories are available to clients alongside application repositories for installations to work.
+
 .Procedure
 . In the {ProjectWebUI}, navigate to *Content* > *Flatpak Remotes*.
 . In the list of Flatpak Remotes, click the name of the remote you want to mirror.
 . In the list of remote repositories, locate the repository you want to mirror.
 . Select the *Mirror Action* menu on the row of the repository.
 . In the *Mirror* window, select the existing product where you want to create the new repository.
-. Click *Mirror*.
+. Click *Mirror*.
```

Comment:
It looks like you're changing the file ending. See https://github.com/theforeman/foreman-documentation/pull/4095

Could you please have a look at your editor settings; I suspect that it's something done by your editor & not by choice.